src/components/ErrorState/index.tsx

```diff
@@ -53,6 +56,8 @@ export const ErrorState = ({
   buttonLabel,
   disabled,
   block,
+  buttonOnClick,
+  buttonisLoading,
```

Comment:
o que tu acha de só `isLoading` ou `isButtonLoading`

Reply:
I fix this in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/39/commits/6e30e89fae38f31f0da895b51fa7e00b44eb891a)